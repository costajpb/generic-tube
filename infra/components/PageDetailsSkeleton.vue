<script setup lang="ts">
import BaseSkeleton from '@/infra/components/BaseSkeleton.vue'
</script>

<template>
  <div :class="classes.container">
    <div>
      <BaseSkeleton :class="classes.title" />
    </div>
    <div>
      <BaseSkeleton :class="classes.content" />
    </div>
  </div>
</template>

<style module="classes" lang="postcss">
.container {
  margin-inline: var(--layout-margin-inline-default);
  margin-block: var(--layout-margin-block-default);
}

.title {
  width: 20ch;
  height: 2em;
}

.content {
  width: 50ch;
  max-width: 70vw;
  height: 5em;
}
</style>
