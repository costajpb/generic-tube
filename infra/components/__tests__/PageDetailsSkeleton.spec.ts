import snapshowWrapper from "@/infra/__tests__/snapshotWrapper"
import PageDetailsSkeleton from "../PageDetailsSkeleton.vue"

describe('infra/components/PageDetailsSkeleton', () => {
    it('should render', () => {
        expect(snapshowWrapper(PageDetailsSkeleton)).toMatchSnapshot()
    })
})