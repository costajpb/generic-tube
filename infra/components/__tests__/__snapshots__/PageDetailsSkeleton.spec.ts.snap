// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`infra/components/PageDetailsSkeleton > should render 1`] = `
<div
  class="_container_9b1f4b"
>
  <div>
    <div
      class="_wrapper_587f33 _title_9b1f4b"
    >
      
      
    </div>
  </div>
  <div>
    <div
      class="_wrapper_587f33 _content_9b1f4b"
    >
      
      
    </div>
  </div>
</div>
`;
